api/routes/participant.py

```diff
@@ -148,3 +158,25 @@ async def get_external_participant_id_to_internal_sample_id_export(
         media_type=export_type.get_mime_type(),
         headers={'Content-Disposition': f'filename={filename}'},
     )
+
+
+@router.post(
+    '/{project}/update-single-participant', operation_id='updateSingleParticipant'
```

Comment:
I think we should infer the `{project}` in the layer just before the write - otherwise this would also allow someone to specify any project they have write access to, even if the participant didn't belong to that project. 

Could we also put the participant ID in the URL, maybe something like

```suggestion
    '/{participant_id}/update', operation_id='updateParticipant'
```